GTBitsOfGood/nextjs-starter
# Luke9kim8/36 fix husky

## Fix Husky pre-commit hook

Issue Number(s): #36 

* Add `.husky/pre-commit` by running `npx install husky`
* `pre-commit` runs `npx lint-staged` before commiting
* If the lint fails, 

### Checklist

### Critical Changes
* Add pre-commit hook

### Related PRs
N/A

### How to Test

1. Try creating a project using the template
2. Write some janky ugly js code
3. Try commiting and see if pre-commt hook runs
